### ISSUE TITLE
git tags are missing for releases

### ISSUE THREAD

**Issue by sam-github** [ibmdb/node-ibm_db#220]
I'm having difficulty correlating npm releases with the code the release came from. This project isn't tagging releases:

Please start tagging your releases, and if you can retro-tag the early releases, that would be helpful.

I strongly recommend using `slt-release` from https://github.com/strongloop/strong-tools#slt-release, it is used by APIConnect, and it will give you consistent conventional releases, with changelogs, git tags, formatted commit messages, etc.

```
node-ibm_db (master u=) % npm info ibm_db .versions
 
[ '0.0.1',
  '0.0.2',
  '0.0.3',
  '0.0.5',
  '0.0.6',
  '0.0.8',
  '0.0.9',
  '0.0.10',
  '0.0.11',
  '0.0.12',
  '0.0.13',
  '0.0.14',
  '0.0.15',
  '0.0.16',
  '0.0.17',
  '0.0.18',
  '0.0.19',
  '1.0.0',
  '1.0.1',
  '1.0.2' ]

node-ibm_db (master u=) % git tg
bd0f0c1a3bed37d5d9be60fb559457c677df25cd refs/tags/1.0.0
878b1029ee303764a0b3b6b959482c5adbabe221 refs/tags/v-0.0.1
48092f543ab82cc70c3f967ab943206073fa8748 refs/tags/v-0.0.2
343ab89179bd75c22e2f4e2023cdd7f268a9e22b refs/tags/v-0.0.3
```

**Comment by bimalkjha** [comment]
We can get the source code for a particular release by just installing that version of ibm_db. Not sure why we should tag the release on git-hub too. We have decided to tag only the major releases, but can think to tag every release. Would explore strong-tools and features provided by it. Thanks.

**Comment by bimalkjha** [comment]
$ npm info ibm_db .time
 
{ modified: '2017-01-17T09:12:06.761Z',
  created: '2014-01-22T10:40:37.711Z',
  '0.0.1': '2014-01-22T10:40:48.784Z',
  '0.0.2': '2014-10-27T10:00:50.034Z',
  '0.0.3': '2014-10-29T13:49:15.937Z',
  '0.0.4': '2014-12-10T10:10:13.512Z',
  '0.0.5': '2014-12-10T10:36:20.957Z',
  '0.0.6': '2014-12-11T08:23:53.920Z',
  '0.0.7': '2015-02-23T10:32:51.067Z',
  '0.0.8': '2015-02-23T10:53:51.817Z',
  '0.0.9': '2015-06-10T18:51:25.531Z',
  '0.0.10': '2015-06-29T17:26:58.159Z',
  '0.0.11': '2015-07-08T20:12:33.682Z',
  '0.0.12': '2015-07-30T16:11:39.928Z',
  '0.0.13': '2015-08-04T09:01:11.691Z',
  '0.0.14': '2015-08-11T19:32:13.572Z',
  '0.0.15': '2015-09-15T16:03:10.725Z',
  '0.0.16': '2016-01-05T07:15:54.462Z',
  '0.0.17': '2016-01-11T16:18:57.835Z',
  '0.0.18': '2016-01-28T17:42:03.123Z',
  '0.0.19': '2016-01-30T10:17:23.079Z',
  '1.0.0': '2016-07-08T13:39:20.769Z',
  '1.0.1': '2016-10-31T20:55:56.450Z',
  '1.0.2': '2017-01-17T09:12:06.761Z' }
$

**Comment by gibfahn** [comment]
It's pretty standard to tag every release on Github, in fact the [npm publish docs](https://docs.npmjs.com/getting-started/publishing-npm-packages) suggest that you do that.

Tagging your releases is also a requirement for testing the module through citgm, which is one of the things we're going for for all IBM modules.

I don't think I've ever seen an npm module that didn't tag their releases (usually it's `v1.0.1` for version `1.0.1`), and it's easy enough to go back and tag them now, so I'd definitely suggest doing that.

It's also really helpful to quickly see the state of the repo at each tag (e.g. what was in the `test/` directory, which shouldn't be published in npm.

**Comment by ibmdb** [comment]
@gibfahn Will do it from next releases. Thanks.

**Comment by sam-github** [comment]
It would helpful (though its a bit tedious), if the git tags could be added retroactively for every version that has been released to npmjs.org

**Comment by bimalkjha** [comment]
Released ibm_db@2.0.0 with proper git-tag. Thanks.
